subcommands/waves/init.go

```diff
@@ -157,10 +158,11 @@ func doInitWave(cmd *cobra.Command, args []string) {
 WARNING: You created a wave for a target version without static deltas.
 
 We recommend that you generate static deltas for your production targets to optimize
-OTA update download. Consider generating a static delta for targets using: 
+OTA update download. 
```

Comment:
```suggestion
We recommend that you generate static deltas for your production targets to optimize OTA update download.
```

The same applies above.